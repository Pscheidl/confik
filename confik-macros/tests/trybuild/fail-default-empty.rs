#[derive(confik::Configuration)]
struct Config {
    #[confik(default = "")]
    _param: String,
}

fn main() {}
