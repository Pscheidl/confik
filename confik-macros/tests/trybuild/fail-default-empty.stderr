error: Unable to parse provided default as rust expression:
 --> tests/trybuild/fail-default-empty.rs:3:24
  |
3 |     #[confik(default = "")]
  |                        ^^
